docs/preview/index.md

```diff
@@ -5,6 +5,8 @@ layout: default
 
 # Installation
 
+* [Show Arcus project templates in Visual Studio](features/show-arcus-templates-in-visualstudio)
```

Comment:
This is not really for installing our templates is it?

Reply:
Hmm, yeah, not really no 😅 . Any suggestions to where it should go? Maybe under just above this title.

Reply:
I would introduce a new "Using our templates" section before installation that:
1. Provides a list of our templates with link to how to configure them
2. has an item called "Using Arcus templates in Visual Studio"
What do you think?

Based on the rendered headers, I would change the ones for every template to an `###` since it's a bit big now imo

Reply:
Done this, feel free to check.